detect_secrets/plugins/keyword.py

```diff
@@ -49,30 +49,31 @@
 BLACKLIST_REGEX = re.compile(
     # Followed by double-quotes and a semi-colon
     # e.g. private_key "something";
+    # e.g. private_key 'something';
     r'|'.join(
-        '(.*?){}(.*?)"(.*?)";'.format(
+        '{}(.*?)("|\')(\\S*?)(\'|");'.format(
             value,
         )
         for value in BLACKLIST
     ) + '|' +
     # Followed by a :
     # e.g. token:
     r'|'.join(
-        '(.*?){}[:](.*?)'.format(
+        '{}:'.format(
```

Comment:
We need to add a capturing group here. Otherwise, we're not going to get anything.